samples/bot-people-picker-adaptive-card/nodejs/Readme.md

```diff
@@ -56,7 +56,7 @@ Make sure you've downloaded and installed Ngrok on your local machine. ngrok wil
 Run ngrok - point to port 3978
 
     ```bash
-    ngrok http -host-header=rewrite 3978
+    ngrok http 3978 --host-header="localhost:3978"
     ```
 
```

Comment:
Why did we deleted these files?
![image](https://user-images.githubusercontent.com/59052547/232966569-d5a44acf-1599-42ef-86e9-e16490a465d6.png)

Reply:
fixed